terraform/shared/statuscake.tf

```diff
@@ -0,0 +1,15 @@
+data "aws_ssm_parameter" "statuscake_api_key" {
+  name = aws_ssm_parameter.statuscake_api_key.name
+}
+
+#provider "statuscake" {
```

Comment:
Given this, can we make sure we document this in the new environment setup (that this will fail until the parameter is set up, but you need to run some terraform to set up SSM)